test/gscraper/accounts/user_test.exs

```diff
@@ -0,0 +1,65 @@
+defmodule Gscraper.Accounts.UserTest do
+  use Gscraper.DataCase
+
+  alias Gscraper.Accounts.User
+
+  describe "changeset" do
+    test "username, password and password_confirmation are required" do
+      changeset = User.changeset(%User{}, %{})
+
+      refute changeset.valid?
+      assert %{username: ["can't be blank"]} = errors_on(changeset)
+      assert %{password: ["can't be blank"]} = errors_on(changeset)
+      assert %{password_confirmation: ["can't be blank"]} = errors_on(changeset)
```

Comment:
We could do 

```elixir
assert errors_on(changeset) == %{
   username: ["can't be blank"],
   password: ["can't be blank"],
   password_confirmation: ["can't be blank"]
}
```

With `==`, we can be sure that changeset contains only these errors. It will errors if there are any other unexpected errors.